src/content/docs/logs/logs-context/get-started-logs-context.mdx

```diff
@@ -101,6 +106,39 @@ Our log ingest pipeline automatically masks credit cards, Social Security number
 
 You can also create custom rules to mask or hash sensitive data in your logs with our obfuscation feature. This is critical when it is impractical or impossible to restrict access to sensitive data, or when some data should never be stored by New Relic. Read our [obfuscation documentation](/docs/logs/ui-data/obfuscation-ui/) to find out more.
 
+## Details about log forwarding [#forwarding-details]
```

Comment:
This looks correct for Node and .NET, it would probably be worth checking with the other teams to find out if there are any other exceptions.